react/src/components/organisms/TabContainer/style.scss

```diff
@@ -0,0 +1,98 @@
+@import "global";
+
+.ma__tab-container, %ma__tab-container {
+  display: flex;
+  flex-direction: column;
+  .ma__tab-container-head {
+    display: inline-flex;
+    height: 60px;
+    flex-wrap: nowrap;
+    overflow-x: auto;
+    -webkit-overflow-scrolling: touch;
+    -ms-overflow-style: -ms-autohiding-scrollbar;
+    .ma__tab-title, %ma__tab-title {
+      border-color: $c-gray-light;
+      outline-color: $c-gray-light;
+      cursor: pointer;
+      padding: 1.5rem 1.5rem 0.8rem 1.5rem;
+      font-size: 1.25rem;
+    }
+    .ma__tab-title:focus, .ma__tab-title--active:focus {
+      outline: none !important;
+      border-color: $c-gray-light !important;
+      border-top-color: $c-primary !important;
+      outline-color: $c-gray-light !important;
```

Comment:
outline color set but outline never applied?

Reply:
Another thought i have here is that instead of border we could use outline so that you don't have to set transparent borders